includes/options/class-amp-options-manager.php

```diff
@@ -319,4 +327,26 @@ public static function persistent_object_caching_notice() {
 			);
 		}
 	}
+
+	/**
+	 * Render the cache miss admin notice.
+	 *
+	 * @return void
+	 */
+	public static function render_cache_miss_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		if ( self::get_option( 'enable_response_caching' ) ) {
+			return;
+		}
+
+		printf(
+			'<div class="notice notice-warning is-dismissible"><p>%s <a href="%s">%s</a></p></div>',
+			esc_html__( "The AMP plugin's response cache disabled due to detecting randomly generated content.", 'amp' ),
+			esc_url( 'https://github.com/Automattic/amp-wp/wiki/Response-cache#automatically-disabling-of-the-response-cache' ),
```

Comment:
@westonruter I opened a [new Wiki page](https://github.com/Automattic/amp-wp/wiki/Response-cache) and started the process of explaining the cache.  Once we get the UI done, then screenshots can be added to the page to provide users/developers more clarity.